airflow/www/security_manager.py

```diff
@@ -290,23 +323,36 @@ def has_access(self, action_name: str, resource_name: str, user=None) -> bool:
 
         Example actions might include can_read, can_write, can_delete, etc.
 
+        This function is called by FAB when accessing a view. See
```

Comment:
Does it men "per-connection" access will not be possible in the UI for non-FAB auth managers (unless we do one of the two options)?

Reply:
Correct :/

Reply:
And it is true for every view auth generated by Flask

Reply:
Then my bet will be on changing the inheritance/decorator. Seems like mostly automate'able and way faster to complete, and if I am not mistaken - we want to get rid of the FAB dependence (and ideally/eventually make it provider dependency not core dependency if possible). Not sure if this would be possible even now, but it should be our end-goal (but maybe I am do not understand the extent of it of course and how tightly coupled it is).

Reply:
No, it is definitely possible :) My wanted to raise attention with this comment, goal achieved :) But I think it should not be done as part of this PR. I'll create an issue for it

Reply:
> No, it is definitely possible :) I wanted to raise attention with this comment, goal achieved :) But I think it should not be done as part of this PR. I'll create an issue for it

Agree

Reply:
This is done in this PR: #35380